x-pack/test/detection_engine_api_integration/security_and_spaces/group1/get_prebuilt_rules_status.ts

```diff
@@ -0,0 +1,134 @@
+        timelines_not_updated: 0,
+      });
+    });
+
+    it('should show that one custom rule is installed when a custom rule is added', async () => {
+      await createRule(supertest, log, getSimpleRule());
+
+      const body = await getPrebuiltRulesStatus(supertest);
+      expect(body).toMatchObject({
+        rules_custom_installed: 1,
+        rules_installed: 0,
+        rules_not_installed: 0,
+        rules_not_updated: 0,
+        timelines_installed: 0,
+        timelines_not_installed: expect.any(Number),
+        timelines_not_updated: 0,
+      });
+    });
+
+    PACKAGE_TYPES.forEach((packageType) => {
+      describe(`package type: ${packageType}`, () => {
+        it('should display the number of rules and templates available to install', async () => {
+          await createPrebuiltRuleAssetSavedObjects(es, packageType);
+          const body = await getPrebuiltRulesStatus(supertest);
+
+          expect(body).toMatchObject({
+            rules_custom_installed: 0,
+            rules_installed: 0,
+            rules_not_installed: RULES_COUNT_BY_PACKAGE_TYPE[packageType],
+            rules_not_updated: 0,
+            timelines_installed: 0,
+            timelines_not_installed: expect.any(Number),
+            timelines_not_updated: 0,
+          });
+
+          expect(body.timelines_not_installed).toBeGreaterThan(0);
+        });
+
+        it('should show rules and timelines are installed when adding pre-packaged rules', async () => {
+          await installPrebuiltRulesFromMocks(supertest, es, packageType);
+
+          const body = await getPrebuiltRulesStatus(supertest);
+          expect(body).toMatchObject({
+            rules_custom_installed: 0,
+            rules_installed: RULES_COUNT_BY_PACKAGE_TYPE[packageType],
+            rules_not_installed: 0,
+            rules_not_updated: 0,
+            timelines_installed: expect.any(Number),
+            timelines_not_installed: 0,
+            timelines_not_updated: 0,
+          });
+
+          expect(body.timelines_installed).toBeGreaterThan(0);
+        });
+
+        it('should show available rule updates', async () => {
+          await installPrebuiltRulesFromMocks(supertest, es, packageType);
+
+          // Increment the version of one of the installed rules
+          await incrementPrebuiltRuleVersion(es, ELASTIC_SECURITY_RULE_ID);
+
+          const body = await getPrebuiltRulesStatus(supertest);
+          expect(body).toMatchObject({
+            rules_custom_installed: 0,
+            rules_installed: RULES_COUNT_BY_PACKAGE_TYPE[packageType],
+            rules_not_installed: 0,
+            rules_not_updated: 1,
+            timelines_installed: expect.any(Number),
+            timelines_not_installed: 0,
+            timelines_not_updated: 0,
+          });
+
+          expect(body.timelines_installed).toBeGreaterThan(0);
```

Comment:
In all these tests, checks for timelines are probably irrelevant, and should be moved to a separate set of tests.

Reply:
I extracted timelines-related tests into a separate file. I didn't work on improving coverage there, though, as that is out of scope.